usbsdmux/i2c_gpio.py

```diff
@@ -25,10 +24,10 @@
 
 class I2cGpio(ABC):
     # Registers inside the supported GPIO expanders
-    _register_inputPort = 0x00
-    _register_outputPort = 0x01
-    _register_polarity = 0x02
-    _register_configuration = 0x03
+    register_inputPort = 0x00
+    register_outputPort = 0x01
+    register_polarity = 0x02
+    register_configuration = 0x03
```

Comment:
This seems to be an unrelated change?

Reply:
In the commit it was on: yes.
In general: Access to the registers was used to access some features of the underlying device.

Not I've added some proper abstraction. Register definitions are now private again.